cancat/uds/utils.py

```diff
@@ -12,24 +12,24 @@
 
 
 def get_uds_29bit_srcid(arbid):
-    consts = uds.ARBID_CONSTS['29bit']
+    consts = uds.ARBID_CONSTS[1]
```

Comment:
please make the "1" into a "constant"  like ARBCONST_11BIT and ARBID_29BIT or something like that.

Reply:
how do these names feel to you?
```
CAN_MSG_11BIT = 0
CAN_MSG_29BIT = 1
```

Reply:
i like them at first blush, but on second thought, the CAN_MSG part seems misleading, no?  it doesn't mention anything about ARBID, and CAN_MSG looks too much like other things in CanCat.  thoughts?

Reply:
That was partially my intention, to have these values match the `extflag` values passed to `CANxmit()` or `UDS()`.  But in retrospect I feel you are correct that it is too similar to the other CAN_MSG_* constants.  I think I like
```
ARBID_11BIT = 0
ARBID_29BIT = 1
```